### DISCUSSION TITLE
Scheduling

### DISCUSSION THREAD

**Discussion by expxx** [NamelessMC/Nameless#2136] [Ideas]
So this is just an example, but still applies, so like you can setup a schedule so if the application has the [Closed] tag, after one hour of no responses, set it to [ThisTag] or whatever.

**Comment by tadhgboyle** [comment]
~~Hi,
Thanks for the idea. Please make an issue here https://github.com/partydragen/Nameless-Forms/issues, as this request is for a third party module.~~

Sorry, I just realized you could have meant the built in Forum module. Can you confirm what you mean by "application"?

**Comment by expxx** [comment]
I meant like in general, not only that forums module, but other things to, like when a user registers or something, just general events.

**Comment by StormyIceLeopard** [comment]
What about if a user does not verify their account in x time it deletes their account.
What about if a user requests a password reset link, after x time of no use expire the link.

**Reply by Derkades** [reply]
Not really related to this, please see https://github.com/NamelessMC/Nameless/issues/1519
